crates/rpc/rpc/src/otterscan.rs

```diff
@@ -120,6 +120,7 @@ where
                             TransferKind::Create => OperationType::OpCreate,
                             TransferKind::Create2 => OperationType::OpCreate2,
                             TransferKind::SelfDestruct => OperationType::OpSelfDestruct,
+                            TransferKind::EofCreate => todo!(),
```

Comment:
map to `OpCreate`?

Reply:
the code here is just formatting an RPC response about what happened on the blockchain, not using the operation in execution. we do need to block or panic.

Reply:
otterscan rpc is used in otterscan block scanner right?

Reply:
returning unimplemented error for now